README.md

```diff
@@ -234,6 +200,22 @@ To build a new release package for foreman project for example, do this:
 
     $ tito release koji-foreman
 
+## Releasing a package
+
+After a PR has been merged to add or update a package, then the package should be released.
+
+1. Switch to the directory for the package to be released, i.e. for the rubygem-apipie-rails package:
+
+    `$ cd packages/foreman/rubygem-mysql2/`
+
+1. Do a scratch build of the release first:
+
+    `$ tito release --scratch koji-foreman koji-foreman-plugins`
+
+1. Verify package built successfully in koji, and then release:
+
+    `$ tito release koji-foreman koji-foreman-plugins`
```

Comment:
Sorry to add a little bit more work. If the goal is for Obal to be our source of truth, we probably want an Obal section here and a 'manual' mode. This should be easy to write though:

```
obal release rubygem-mysql2
```

Reply:
@ehelms I know this PR has been open a bit, but iirc the intent with this unit of work was to just use the tito builder that is in obal, not re-define the workflow to use obal yet.

Reply:
That's fair, we can leave the full switch to Obal for tooling as a separate change.